include/znc/Modules.h

```diff
@@ -356,7 +359,7 @@ class CModCommand {
 class CModule {
 public:
 	CModule(ModHandle pDLL, CUser* pUser, CIRCNetwork* pNetwork, const CString& sModName,
-			const CString& sDataDir);
+			const CString& sDataDir, CModInfo::EModuleType eType = CModInfo::NetworkModule);
```

Comment:
Gah, I don't want to touch them... I got some strange type errors for the enum argument. This is compatible since it has the default value.